week-1/Homework/extra/1-currency-conversion.js

```diff
@@ -16,20 +19,24 @@ function convertToUSD() {}
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToBRL() {}
+function convertToBRL(sterlingPrice) {
+  const brlPrice = sterlingPrice * 5.7;
+  const finalPrice = brlPrice + brlPrice / 100;
+  return finalPrice;
+}
```

Comment:
I find the names of the variables very descriptive 👌

Notice though that, with this implementation, the 1% fee we are supposed to be applying is actually a 1% _gift_ 😱

Reply:
@frosas Really appreciate you reviewing my code :D... I checked it again, I think you are right (I should probably deduct the fee from the final BRL price), but if I change the code the test will fail. Do you think the test is incorrect?

Reply:
Oh I see. I'd say the test is incorrect as it just makes no sense as it is, but feel free to leave this as it is in any case.